src/mdx-pages/blog/videojs-8-and-vhs-3.mdx

```diff
@@ -0,0 +1,163 @@
+
+Overall, the minfied and gzipped size of VHS is **reduced by around 4%**. More savings are likely available through further removal of dead code.
+
+### Browser/Device Support
+
+With these releases, our new browser/device targets are generally focused on the last 3 major versions of so-called "evergreen" browsers:
+
+* Chrome
+* Firefox
+* Safari
+* Edge (Chromium, not Legacy)
+
+However, with Video.js being used in more contexts than just desktop and mobile web, like Smart TVs or OTT devices, we have an additional minimum support threshold for _Chromium-based browsers version 53 and newer_. This grants support for:
+
+* [webOS TV](https://webostv.developer.lge.com/discover/specifications/web-engine/) 4.x and newer
+* [Tizen](https://developer.samsung.com/smarttv/develop/specifications/web-engine-specifications.html) 2018 models and newer
+
+## New Video.js `TitleBar` Component
+
+The new `TitleBar` component will show a UI element across the top of the player which displays the title and/or description of the current media in the player. The `TitleBar` will not show if no title or description is provided.
+
+### Using `loadMedia` to Populate the `TitleBar`
+
+The easiest way to provide a title and/or description is to use your player's `loadMedia` method:
+
+```js
+player.loadMedia({
+  artist: 'Disney',
+  album: 'Oceans',
+  title: 'Oceans',
+  description: 'Journey in to the depths ... and race with dolphins at play.',
+  poster: 'https://vjs.zencdn.net/v/oceans.png',
+  src: [{
+    src: 'https://vjs.zencdn.net/v/oceans.mp4',
+    type: 'video/mp4'
+  }, {
+    src: 'https://vjs.zencdn.net/v/oceans.webm',
+    type: 'video/webm'
+  }]
+})
+```
+
+If you're not familiar with the `loadMedia` method, it's a way to provide additional metadata for your media beyond what is available through the `src` method alone. As you see in the above example, the `title` and `description` are provided and will be used to populate the title bar.
+
+### Populating the `TitleBar` Directly
+
+The `TitleBar` can also be populated through direct input using the component's `update` method:
+
+```js
+player.titleBar.update({
+  title: 'Oceans',
+  description: 'Journey in to the depths ... and race with dolphins at play.'
+});
+```
+
+The title and/or description can be removed by passing an empty string for one or both of these values:
+
+```js
+player.titleBar.update({
+  title: '',
+  description: ''
+});
+```
+
+If both are removed, the `TitleBar` will no longer be visible.
+
+## New Video.js Utility Objects
+
+Over time, the `videojs` namespace has become cluttered with a wide variety of utility functions. One of the changes we are introducing is a more deliberately designed interface for these utility functions.
+
+The guiding principle here was that if a function didn't feel like a core part of the library, but was still _potentially useful_, we exposed it as part of a utility object instead of a top-level function.
```

Comment:
Do we want to qualify this a little as useful for plugins and code interacting with Video.js. i.e. we are not purporting to be a general purpose utlity library?

Reply:
Yes, I think that it's worth mentioning. "Potentially useful to plugins" or something.